tests/kernel/install_ltp.pm

```diff
@@ -63,13 +66,15 @@ sub install_dependencies {
 sub install_from_git {
     my $url = get_var('LTP_GIT_URL') || 'https://github.com/linux-test-project/ltp';
     my $tag = get_var('LTP_RELEASE') || '';
+    my $configure = './configure --with-open-posix-testsuite --with-realtime-testsuite';
+    my $extra_flags = get_var('LTP_EXTRA_CONF_FLAGS') || '';
```

Comment:
JFYI get_var function accept second parameter as default value so same thing could be written like 
`get_var('LTP_EXTRA_CONF_FLAGS',''); `
but it is just matter of taste :) your way is ok too